### ISSUE TITLE
Failed to find mapping to ipv6_address, protocol TCP

### ISSUE THREAD

**Issue by vladns** [miniupnp/miniupnp#669]
What settings should be in order for ipv6 support to work? In my config there are the following settings regarding ipv6:
```
ext_ifname=ppp0
ipv6_disable=no
```
The systemd unit has the following settings (fragment regarding ipv6):
```
EnvironmentFile=/etc/miniupnpd/miniupnpd.conf
ExecStartPre=/etc/miniupnpd/iptables_init.sh -i $ext_ifname
ExecStartPre=/etc/miniupnpd/ip6tables_init.sh -i $ext_ifname
ExecStopPost=/etc/miniupnpd/iptables_removeall.sh -i $ext_ifname
ExecStopPost=/etc/miniupnpd/ip6tables_removeall.sh -i $ext_ifname
```
In the log I see this message:
```
oct 08 12:34:35 router systemd[1]: Started Lightweight UPnP IGD daemon.
oct 08 12:34:35 router miniupnpd[42373]: HTTP listening on port 46071
oct 08 12:34:35 router miniupnpd[42373]: HTTP IPv6 address given to control points : [322:33a:75e4:72d4::1]
oct 08 12:34:35 router miniupnpd[42373]: Listening for NAT-PMP/PCP traffic on port 5351
oct 08 12:35:45 router miniupnpd[42373]: Failed to remove PCP mapping to 192.168.2.122:11148 TCP
oct 08 12:35:45 router miniupnpd[42373]: Failed to find mapping to 322:33a:75e4:72d4:ae45:d2ef:da42:a4c2:11148, protocol TCP
oct 08 12:35:45 router miniupnpd[42373]: Failed to remove PCP mapping to 192.168.2.125:11148 UDP
oct 08 12:35:45 router miniupnpd[42373]: Failed to find mapping to 322:33a:75e4:72d4:ae45:d2ef:da42:a4c2:11148, protocol UDP
oct 08 12:35:45 router miniupnpd[42373]: Failed to find mapping to 2a04:2cd3:2ed5:da32:e135:ec34:edda:234a:11148, protocol TCP
oct 08 12:35:45 router miniupnpd[42373]: Failed to find mapping to 322:33a:75e4:72d4:ae45:12ef:4a42:a4c2:11148, protocol TCP
oct 08 12:35:45 router miniupnpd[42373]: Failed to find mapping to 322:33a:75e4:72d4:ae45:12ef:4a42:a4c2:11148, protocol UDP
oct 08 12:35:45 router miniupnpd[42373]: Failed to find mapping to 2a04:2cd3:2ed5:da32:e135:ec34:edda:234a:11148, protocol UDP
oct 08 12:35:45 router miniupnpd[42373]: Failed to find mapping to 2a04:2cd3:2ed5:da32:e135:ec34:edda:234a:11148, protocol TCP
oct 08 12:35:45 router miniupnpd[42373]: Failed to find mapping to 2a04:2cd3:2ed5:da32:e135:ec34:edda:234a:11148, protocol UDP
```
Archlinux system.

**Comment by miniupnp** [comment]
Well it looks like the ipv6 support works.

What are you testing ? Trying to remove or get information on non existing mappings will indeed produce failure messages.

**Comment by vladns** [comment]
I'm using qbittorrent-nox on nas. It worked once or twice, then I noticed that the iptables rules were not added after restarting qbittorrent-nox. Each restart was accompanied by a message in the log, but no iptables rules were added. So I decided to ask. Now, after updating all systems (router and nas) and rebooting, the rules were added again, but the message still appeared in the log. I just stopped qbittorrent-nox, the rules were deleted, after starting qbittorrent-nox the rules appeared. There are no new error messages in the log. Now it turns out that everything seems to be working.
